packages/matter.js/src/cluster/WindowCoveringCluster.ts

```diff
@@ -0,0 +1,177 @@
+/**
+ * @license
+ * Copyright 2022 The matter.js Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { TlvEnum, TlvUInt16 } from "../tlv/TlvNumber.js";
+import { TlvObject, TlvOptionalField } from "../tlv/TlvObject.js";
+import { Attribute, ClusterExtend, Command, OptionalCommand, TlvNoResponse } from "./Cluster.js";
+import { ClusterServerObjForCluster } from "./server/ClusterServer.js";
+import { WindowCoveringClusterSchema } from "./schema/WindowCoveringCluster.js";
+import { MatterApplicationClusterSpecificationV1_0 } from "../spec/Specifications.js";
+
+/** alias for percentages expressed as 0 to 100.00 with 2 decimals */
+const WCPercent100ths = TlvUInt16.bound({ max: 10000 });
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 5.3.5.1 */
+export enum WindowCoveringType {
```

Comment:
Yes normally yes, but because we stiill have some CLusterExtdn here (to not block vves until generation is there) they are needed, but I do not wanted to export them in the other one because they should not be needed to be exported. And because we would throw that file away anyway when generation is there I decided for the "copy" way.

Reply:
Cool I just need to avoid accidentally deleting.